pkg/remote/builtin/github/github.go

```diff
@@ -34,12 +37,14 @@ type GitHub struct {
 
 func New(conf *config.Config) *GitHub {
```

Comment:
Well, i ported the driver system from store, so we need some corrections in specs ( http://readme.drone.io/docs/setup/#/github.md )

need to add `DRONE_GITHUB_API`, if we use gtihub hosted solution, api placed on subdomain, by default we can detect this if `DRONE_GITHUB_HOST` specified, but `DRONE_GITHUB_API` not, also we can show warning for this.